releases/v0.2.0.md

```diff
@@ -0,0 +1,17 @@
+# Release Notes for v0.2.0
+
+## Improvements
+
+- Added an end-to-end test flow after releasing new versions to enhance system robustness.
+- Optimized the UI during answer retrieval to provide better feedback on current processes.
+- Support for connecting to Self-Managed TiDB clusters that also have vector search capabilities.
+- Integrated support for the BaiSheng reranker.
+
+## Bug Fixes
+
+- Fixed an issue when creating synopsis entity nodes.
+- Fix the delete button in LLM/Reranker list page.
+- Fixed a SiteSetting cache issue that prevented proper synchronization across processes.
+
+
+If you are deploying tidb.ai using docker, please refer to this [document](https://tidb.ai/docs/deploy-with-docker#upgrade) for upgrading your tidb.ai.
```

Comment:
How about moving this section to https://github.com/pingcap/tidb.ai/blob/main/frontend/app/src/pages/docs/changelog.mdx

Reply:
@wd0517

Reply:
The changelog page should be removed as it duplicates with the release note.